apps/bfd-model/bfd-model-rda/src/main/java/gov/cms/bfd/model/rda/RdaApiClaimMessageMetaData.java

```diff
@@ -0,0 +1,84 @@
+package gov.cms.bfd.model.rda;
+
+import java.time.Instant;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.FieldNameConstants;
+
+@Entity
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+@FieldNameConstants
+@Table(name = "`RdaApiClaimMessageMetaData`", schema = "`pre_adj`")
+public class RdaApiClaimMessageMetaData {
+  private static final String SequenceName = "`rda_api_claim_message_meta_data_meta_data_id_seq`";
+
+  /**
+   * We insert a record every time a change is received from RDA API so the primary key has to be
+   * unique even if we receive the same message twice.
+   */
+  @Id
+  @Column(name = "`metaDataId`", nullable = false, updatable = false)
+  @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = SequenceName)
+  @SequenceGenerator(
+      name = SequenceName,
+      schema = "`pre_adj`",
+      sequenceName = SequenceName,
+      allocationSize = 25)
+  @EqualsAndHashCode.Include
+  private long metaDataId;
+
+  /** Either FISS or MCS. */
+  @Enumerated(EnumType.STRING)
+  @Column(name = "`claimType`", length = 4, nullable = false)
```

Comment:
Thinking about how many rows this table will have, would it be best ot just use 1 char here.  `F` or `M`?

Reply:
I have converted this to `char(1)` now.  Had to make the field a `Character` because `@Enumerated` doesn't work with `char(1)` columns.  It gives a hibernate validation error.